tests/a_scripts/train/test_train.py

```diff
@@ -11,12 +11,12 @@
 from omegaconf import DictConfig, ListConfig
 from omegaconf import OmegaConf as om
 
+from llmfoundry.train import TrainConfig  # noqa: E402
```

Comment:
What is E402?

Reply:
Not sure, was there when I started, all I did was change from scripts.train to llmfoundry.train